distribution/Directory.Build.props

```
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <!-- Solution wide properties -->
+  <PropertyGroup Label="Assembly Naming">
+    <Company>CompanyName</Company>
+    <Authors>CompanyName</Authors>
+    <Copyright>© $(Company). All rights reserved.</Copyright>
+    <NeutralLanguage>en</NeutralLanguage>
+    <DefaultLanguage>en-US</DefaultLanguage>
+  </PropertyGroup>
+
+  <PropertyGroup Label="Compile settings">
+    <Nullable>enable</Nullable>
+    <LangVersion>9.0</LangVersion>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <NoWarn>1573,1591,1712</NoWarn>
+
+    <!-- Used by code coverage -->
+    <DebugType>full</DebugType>
+    <DebugSymbols>true</DebugSymbols>
+  </PropertyGroup>
+
+  <PropertyGroup Label="Analyzer settings">
+    <AnalysisMode>AllEnabledByDefault</AnalysisMode>
```

```suggestion
    <AnalysisMode>AllEnabledByDefault</AnalysisMode>
		<EnableNETAnalyzers>true</EnableNETAnalyzers>
		<AnalysisLevel>latest</AnalysisLevel>
		<EnforceCodeStyleInBuild>true</EnforceCodeStyleInBuild>    
```

Lets turn everything on